testing/cloud.py

```diff
@@ -47,6 +47,20 @@ def __getattr__(self, name):
         return os.environ[name]
 
 
+def get_resource_path(resource, local_path):
```

Comment:
Any particular reason you didn't stick with the same method signature as `resource_path`? eg with the variable-length `*resource`?

Reply:
TWEAKING 🎼BEATS #TRIXMIXING1AI ♥️

Reply:
TRIXMI@trixmixing1ai.com